Testing/SystemTests/tests/analysis/QENSGlobalFitTeixeiraWaterSQE.py

```diff
@@ -74,9 +75,20 @@ def runTest(self):
         # Validate
         self._success = True
         try:
-            self.assertTrue(chi2 < 1)
+            self.assertTrue(chi2 < 1)  # goodness of fit
             self.assertTrue(abs(params.row(6)["Value"]-1.58)/1.58 < 0.1) # optimal DiffCoeff
             self.assertTrue(abs(params.row(7)["Value"] - 1.16) / 1.16 < 0.1)  # optimal Tau
+            # check curves are correctly generated by calculating their Chi^2
+            residuals = np.empty(0)
+            for i in range(curves.size()):
+                curveset = curves[i]  # contains data, model, and residuals
+                dataY = curveset.dataY(0)
+                dataE = curveset.dataE(0)
+                modelY = curveset.dataY(1)
+                residuals = np.append(residuals, ((dataY-modelY)/dataE)**2)  # don't trust residuals of curveset
+            otherChi2 = residuals.sum()/len(residuals)
+            print otherChi2, chi2, abs(otherChi2-chi2)/chi2
```

Comment:
Can you fix the print statement for python 3, this systemtest previously passed with python 3 and we don't want to go backwards.

And add `from __future__ import (absolute_import, division, print_function)` to the top.